### ISSUE TITLE
Review lab10 by Abdelouahab Moubane s305716

### ISSUE THREAD

**Issue by AbdelouahabMoubane** [ivanmag22/computational-intelligence#6]
Hi this my review about you Lab 10.
The work is great and complete, you used different approaches, one with Monte Carlo simulations and others with Q-Learning. I appreciated the explanations and plots. I have some suggestions, I think considering a discount factor of 0.6 is low and therefore you don't care much about future premiums, infact if we consider the sequence of powers 0.600, 0.360, 0.216, 0.1296, 0.077, already the premiums between three successive periods weights very little, it's better use a discount factor closer at one.  Another observatin, the learning rate, it's better think it like a portion of learnig, a coefficient of che convex combination like a following formula:
Q(s,a) ← (1-alpha) * Q(s,a) + alpha * (instant_payoff + discount * maxQ(s',a')).
Over all, great job and good luck.

**Comment by ivanmag22** [comment]
Hi,
thank you for the review.
I'll evaluate better the hyper-parameters to use. You gave me good hints.
Have a good luck you too!
